searchapp/templates/layout.html

```diff
@@ -9,7 +9,7 @@
 	<link rel="stylesheet" type="text/css" href="{% static 'pnotify.custom.min.css' %}"/>
 	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.14/semantic.css"/>
     <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
-    <script type="text/javascript" src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
+    <script type="text/javascript" src="http://code.jquery.com/jquery-1.11.2.min.js"></script>
```

Comment:
@sandy9999 I thought this would be a problem? We are serving https and requesting resources from http?